hoodie-client/src/main/java/com/uber/hoodie/WriteStatus.java

```diff
@@ -48,6 +51,16 @@
   private long totalRecords = 0;
   private long totalErrorRecords = 0;
 
+  private final double failureFraction;
+  private final boolean trackSuccessRecords;
+  private final transient Random random;
+
+  public WriteStatus(Boolean trackSuccessRecords, Double failureFraction) {
```

Comment:
A candidate for adding to release notes as all custom WriteStatus now have to take these 2 args

Reply:
any reason to user "Boolean" / "Double" over "boolean" / "double"?

Reply:
reflection does not pick it up otherwise :/